packages/app/src/components/Account.vue

```diff
@@ -118,6 +125,27 @@ const breadcrumbItems = computed((): BreadcrumbItem[] | [] => {
 const transactionsSearchParams = computed(() => ({
   address: props.account?.address,
 }));
+
+const { fetchSNSName } = useSns();
+const snsName = ref<string | null>(null);
+
+watch(
+  () => props.account?.address,
+  async (address) => {
+    snsName.value = null;
+    if (address) {
+      const name = await fetchSNSName(address);
+      if (name) {
+        snsName.value = `${name}.soph.id`;
+      } else {
+        snsName.value = null;
+      }
+    } else {
+      snsName.value = null;
+    }
```

Comment:
```suggestion
    if (address) {
      const name = await fetchSNSName(address);
      if (name) {
        snsName.value = `${name}.soph.id`;
      }
    }
```
Don't think you need the extra value = null case it's already initialized as null right?